spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe 'User Stories' do
+
+  let(:dish1) {Dish.new(name: :pizza, price: 1)}
+  let(:dish2) {Dish.new(name: :chicken, price: 2)}
+  let(:dish_list) {DishList.new([dish1, dish2])}
+  let(:order_list) {OrderList.new(dish_list)}
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+
+
+
+  describe '#prices' do
+    it 'gives a list with the price of each dish' do
+      expect(order_list.prices).to eq ({pizza: 1, chicken: 2})
+    end
+  end
+
+
+  describe '#place_order' do
+
+    # As a customer
+    # So that I can order the meal I want
+    # I would like to be able to select some number of several available dishes
+
+
+    context 'when a selected dish is unavailable' do
+      it 'raises an error' do
+        expect{order_list.place_order({apple: 2})}.to raise_error "An ordered dish is unavailable"
+      end
+    end
+
+    context 'when all selected dishes are available' do
+
+      it 'does not raise an error' do
+        expect{order_list.place_order({pizza: 1})}.not_to raise_error
+      end
+      # As a customer
+      # So that I can verify that my order is correct
+      # I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+      context 'when given total does not match sum of dishes in customer\'s order' do
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)